### ISSUE TITLE
Attempted to add mapping "ScreenShot" twice to trigger

### ISSUE THREAD

**Issue by stephengold** [jMonkeyEngine/jmonkeyengine#1421]
If you attach a `ScreenshotAppState`, detach it, and attach it again, you will get a warning:
```text
WARNING: Attempted to add mapping "ScreenShot" twice to trigger.
```

Here's what I think is happening: `ScreenshotAppState.initialize()` adds a mapping to the `InputManager`. Its `cleanup()` method should remove the mapping, but that method isn't implemented anywhere.

**Comment by MeFisto94** [comment]
You are right but the issue is even worse. If I a not mistaken there is no cleanup method at all, meaning not only will this AppState instance leak due to being registered as a Listener, it also doesn't cleanup the Processor, causing dirty behavior.

**Comment by pspeed42** [comment]
Some of these earlier app states were very bad with life cycle.  Not all of them have been fixed.

**Comment by stephengold** [comment]
Solved by PR #1421 in master at a3c3678

**Comment by stephengold** [comment]
I confirmed that the issue is resolved in v3.4.0-beta3.
